addon/components/new-bourbon-select-field.js

```diff
@@ -102,7 +102,15 @@ export default Component.extend({
         'value',
         this.get('value')
       );
-      return this.get('prompt') ? this.get('internalContent').indexOf(valueHolder) : 0;
+
+      let index = this.get('internalContent').indexOf(valueHolder);
+
+      if (index !== -1) {
+        return index;
+      } else if (index === -1 && !this.get('prompt')) {
+        // default to first option when there is no prompt passed
+        return 0;
+      }
```

Comment:
What's the behavior if `index === -1` but there is a prompt?

Reply:
it will show the prompt...

if `index === -1`, then selectedIndex is `undefined` which causes `selection` to be null, which then causes `label` to show the prompt.  

i am going to add a default to make `selectedIndex` null for this case so that is is a bit cleaner than how it works now.